src/meta/src/manager/cluster.rs

```diff
@@ -112,10 +113,13 @@ where
                     .generate::<{ IdCategory::Worker }>()
                     .await? as WorkerId;
 
-                // Generate parallel units.
                 let parallel_units = self
-                    .generate_cn_parallel_units(worker_node_parallelism, worker_id)
+                    .generate_cn_parallel_units(
+                        property.worker_node_parallelism as usize,
+                        worker_id,
+                    )
                     .await?;
+                let property = self.parse_property(r#type, property);
```

Comment:
Seems the `property` is only meaningful for compute nodes. Should we make it `None` for other worker node types?